checks/evaluation/packaging.go

```diff
@@ -51,7 +51,7 @@ func Packaging(name string, dl checker.DetailLogger, r *checker.PackagingData) c
 
 	if pass {
 		return checker.CreateMaxScoreResult(name,
-			"publishing workflow detected")
+			"published package detected")
```

Comment:
"packaging detected"  since it's the packaging check?
WDUT?

Reply:
not sure :)